src/modules/wizeng/WizengModule.v3

```diff
@@ -53,6 +53,25 @@ class WizengModule extends HostModule("wizeng") {
 				var imported_sig = Canon.sigPR(SigCache.arr_ii, [rtype]);
 				result[j] = HostFunction.new("new_funcref", imported_sig, func);
 			}
+			if (!found && Strings.equal("puts_array", i.imp.field_name)) {
```

Comment:
How about just `puta`? It makes it a little more symmetric with the other put methods.

Reply:
That is fine with me, fixed.